packages/block-library/src/image/deprecated.js

```diff
@@ -632,9 +632,6 @@ const v6 = {
 	},
 	supports: {
 		anchor: true,
-		behaviors: {
-			lightbox: true,
-		},
```

Comment:
This one I'm not 100% sure about. 

I think that we can safely remove the reference to behaviors in those deprecations because if a user does have a block with the `behaviors: { ...}` attribute for some reason, it should just be ignored. 

Could you confirm that @gziolo ?

Reply:
Block supports can be safely removed. The only question is what will happen when there is a value for the block attribute set in the saved HTML as part of the comment metadata.

Reply:
We should probably add a block deprecation for this either way, right? If there is no validation error, you should be able to set an `isElegible` method on the deprecation based on the presence of the top-level `behaviors` attribute. And then the `behaviors` attribute can be migrated to the `lightbox` attribute with the `migrate` method.

This would save having to add the deprecation warning in https://github.com/WordPress/gutenberg/pull/54071/files#diff-a14b61c874095649b236b78413e805b4541277b24eb71ce350a17d9f2b6396b1R765-R779.

Reply:
> One way to test whether renaming the attribute is safe, would be pasting the old saved Image block with the lightbox enabled locally and checking whether the block editor won't trigger the block validation error.

I've tested this by adding the JSON manually to the block and there's no error, so it looks like renaming the attribute is safe.

> We should probably add a block deprecation for this either way, right?

@ajlende This sounds right.

As per [this comment](https://github.com/WordPress/gutenberg/pull/54071#discussion_r1318511575), it looks like we can handle the migration in a separate PR later, right? 

We'll be offering backwards compatibility for 2 Gutenberg releases. As this PR is already pretty big, I think it makes sense to just push this forward and handle the deprecation separately.

Reply:
> As per https://github.com/WordPress/gutenberg/pull/54071#discussion_r1318511575, it looks like we can handle the migration in a separate PR later, right?
>
> We'll be offering backwards compatibility for 2 Gutenberg releases. As this PR is already pretty big, I think it makes sense to just push this forward and handle the deprecation separately.

[Responded below.](https://github.com/WordPress/gutenberg/pull/53851#issuecomment-1712082579)